src/ui-kit/molecules/CustomAlert/CustomAlert.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react"
+
+import "./CustomAlert.scss"
+
+type Props = {
+    action?: String,
+    content?: String,
+    legend?: String
```

Comment:
let's use typescript `string` instead of javascript `String` object here:
```suggestion
    action?: string,
    content?: string,
    legend?: string
```

Reply:
Solved thanks